roboteam_ai/src/stp/plays/referee_specific/PenaltyUsPrepare.cpp

```diff
@@ -23,15 +23,15 @@ PenaltyUsPrepare::PenaltyUsPrepare() : Play() {
         std::make_unique<role::Formation>(role::Formation("formation_8"))};
 }
 
-uint8_t PenaltyUsPrepare::score(const rtt::world::Field& field) noexcept {
+uint8_t PenaltyUsPrepare::score(const rtt::Field& field) noexcept {
     /// List of all factors that combined results in an evaluation how good the play is.
     scoring = {{PlayEvaluator::getGlobalEvaluation(eval::PenaltyUsPrepareGameState, world), 1.0}};
     return (lastScore = PlayEvaluator::calculateScore(scoring)).value();  // DONT TOUCH.
 }
 
 void PenaltyUsPrepare::calculateInfoForRoles() noexcept {
     // We need at least a keeper, and a kicker positioned behind the ball
-    stpInfos["keeper"].setPositionToMoveTo(Vector2(field.getOurGoalCenter()));
+    stpInfos["keeper"].setPositionToMoveTo(Vector2(field.leftGoalArea.rightLine().center()));
```

Comment:
Hmmm, I guess you might have a point. In my opinion, the `leftGoalArea.rightLine().center()` is more clear than the `getOurGoalCenter()`, because to me it was unclear whether 'center' meant the center of the goal area or the center of the goal line, but I do see how that might differ per person. I will again leave that up to the current team.

Reply:
I prefer the `leftGoalArea.rightLine().center()`, but as Floris said this personal preference. I also think keeping the old function reduces the readability of the Field class, which is the opposite of what we want.